test-cases/longevity/longevity-harry-2h.yaml

```diff
@@ -1,5 +1,5 @@
 test_duration: 240
-stress_cmd: ["/usr/bin/cassandra-harry -run-time 2 -run-time-unit HOURS"]
+stress_cmd: ["cassandra-harry -run-time 2 -run-time-unit HOURS"]
```

Comment:
i think you should run it with the full path, like it was before --> `/usr/bin/cassandra-harry`
as you have anyway created a symlink to it

Reply:
This code was failed because the command contains full path
```
        if 'cassandra-harry' in self.params.list_of_stress_tools:
            if not node.is_cassandra_harry_installed:
                node.install_cassandra_harry()
```

Reply:
if it failed because it had the full path, it means that the following line failed:
```
self.remoter.sudo(f"ln -svf {cassandra_harry_tool_path} /usr/bin/cassandra-harry", ignore_status=True)
```

Reply:
`list_of_stress_tools` doesn't hold full paths, it's like that now: ['cassandra-stress', '/usr/bin/cassandra-harry']
so `'cassandra-harry' in self.params.list_of_stress_tools` doesn't work

Reply:
i see it now..